go.mod

```diff
@@ -2,9 +2,4 @@ module github.com/jhump/gopoet
 
 go 1.13
 
-require (
-	github.com/gordonklaus/ineffassign v0.0.0-20200309095847-7953dde2c7bf // indirect
-	github.com/nishanths/predeclared v0.0.0-20200524104333-86fad755b4d3 // indirect
-	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
-	honnef.co/go/tools v0.0.1-2020.1.4 // indirect
-)
+require github.com/google/go-cmp v0.5.6
```

Comment:
Please don't commit these removals. This is how the versions are pinned for the static checkers used in CI.

Reply:
Done.